src/pages/monitor/Monitor.tsx

```diff
@@ -107,11 +109,19 @@ const Endpoint: React.FC<Endpoint> = ({ url, locationEndPoint, disable }) => {
   )
 }
 
-type IsItUp = {
+export type IsItUp = {
   statusIsItUp: string
+  fromMonitor?: boolean
+  url?: string
 }
 
-const IsItUp: React.FC<IsItUp> = ({ statusIsItUp }): JSX.Element => {
+export const IsItUp: React.FC<IsItUp> = ({
+  statusIsItUp,
+  fromMonitor,
+  url,
+}): JSX.Element => {
+  const width = useWindowWidth()
+  const canNavigate = width < 1200 && fromMonitor
```

Comment:
Should we make the endpoint page available on all devices?

Reply:
This page is supposed to complete the information that can't appear on the mobile or tablet version, we can change this too. But should the design be the same on web?

Reply:
lets keep it to mobile and tablet for now 👍